src/components/CommentList/CommentList.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+
+type Props = {
+  comment: Comment[],
+};
+
+export const CommentList: React.FC<Props> = ({ comment }) => (
+  <ul>
+    <h4>Comments:</h4>
+    <CommentInfo comment={comment} />
+  </ul>
```

Comment:
```suggestion
  <h4>Comments:</h4>
  <ul>
    <CommentInfo comment={comment} />
  </ul>
```
В ul мають бути лише li